sanic_jwt/initialization.py

```diff
@@ -0,0 +1,174 @@
+
+        for endpoint in endpoint_mappings:
+            self.__add_single_endpoint(*endpoint)
+
+        self.bp.exception(exceptions.SanicJWTException)(
+            self.response.exception_response)
+
+        if not self.instance_is_blueprint:
+            self.instance.blueprint(
+                self.bp, url_prefix=config.url_prefix)
+
+    def __add_class_views(self):
+        """
+        Include any custom class views on the Sanic JWT Blueprint
+        """
+        config = self.config
+        if 'class_views' in self.kwargs:
+            class_views = self.kwargs.pop('class_views')
+
+            for route, view in class_views:
+                if issubclass(view, HTTPMethodView) and isinstance(route, str):
+                    self.bp.add_route(
+                        view.as_view(),
+                        route,
+                        strict_slashes=config.strict_slashes
+                    )
+                else:
+                    raise exceptions.InvalidClassViewsFormat()
+
+    def __check_initialization(self):
+        """
+        Confirm that required parameters were initialized and report back exceptions
+        """
+        config = self.config
+        if hasattr(config, 'refresh_token_enabled') and \
+            getattr(config, 'refresh_token_enabled') and (
+            not self.kwargs.get('store_refresh_token') or
+            not self.kwargs.get('retrieve_refresh_token')
+        ):
+            raise exceptions.RefreshTokenNotImplemented
+
+        # TODO:
+        # - Add additional checks
+
+    def __initialize_instance(self):
+        """
+        Take any predefined methods/handlers and insert them into Sanic JWT
+        """
+        # Initialize instance of the Authentication class
+        config = self.config
+        self.instance.auth = self.authentication_class(self.app, config=config)
+
+        if 'authenticate' not in self.kwargs:
+            raise exceptions.AuthenticateNotImplemented
+
+        for handler in handlers:
+            handler_name, _ = handler
+            if handler_name in self.kwargs:
+                method = self.kwargs.pop(handler_name)
+                setattr(self.instance.auth, handler_name, method)
+
+    def __load_configuration(self):
+        """
+        Configure settings for the instance in the following order:
+
+        1. Sanic JWT's defaults
+        2. Custom Configuration class
+        3. Key word arguments passed to Initialize
+        """
+        config_to_enable = [x for x in handlers if x[1]]
+        for config_item in config_to_enable:
+            if config_item[0] in self.kwargs:
+                list(map(lambda x: self.kwargs.update(
+                    {x: True, config_item[0]: self.kwargs.get(config_item[0])}),
+                    config_item[1]))
+
+        self.config = self.configuration_class(self.app.config, **self.kwargs)
+
+    def __load_response(self):
+        response = self.response_class()
+        make_response(response)
```

Comment:
Right, I see here what you've told me about the `make_response`, working with references ... I'll check the `make_response` method and add my comments there.